src/components/ProductCard.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <div class="container-card-products">
+    <div class="card-product">
+        <img src="@/assets/images/Tshirt.jpg" class="card-product__image" />
+        <div class="cardProducts__infos">
+            <span class="card-product__title">Camisa de mangas na cor azul</span>
+            <div class="card-product__price">
+                <span class="card-product__real-price">R$45,00</span>
+                <span class="card-product__old-price">R$55,00</span>
+            </div>
+        </div>
+        <div class="card-btn">
+        <button type="submit" class="card-product__add-to-cart">Adicione ao carrinho</button>
+        </div>
+    </div>
+    </div>
+</template>
+
+<style scoped>
+
+
+
+.card-product {
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: rgb(255, 248, 248);
+    width: 215px;
+    height: 400px;
+    justify-content: space-around;
+    box-shadow: 3px 0px 0px -2px rgba(0, 0, 0, 0.25);
+    border-radius: 9px;
+}
+
+.cardProducts__infos {
+    font-size: 14px;
+    font-weight: 500;
+    text-align: center;
+    line-height: 2;
+    padding-right: 20px;
+    padding-left: 20px;
+}
+
+.card-product__image {
+    width: 100%
+}
+
+.card-product__title {
+    margin-top: 22px;
+}
+
+.card-product__price {
+    display: flex;
+    gap: 10px;
+    font-weight: 600;
+    justify-content: center;
+}
+
+.card-product__old-price {
+    font-size: 12px;
+    color: #b5aec4;
+    text-decoration: line-through;
+}
+
+.card-product__real-price {
+    color: #fe4c50;
+    font-weight: 600;
+}
+
+
+.card-product__add-to-cart {
+    border: none;
+    height: 50px;
+    width: 215px;
+    background-color: #FE4C50;
+    font-family: Poppins;
```

Comment:
[???] O nome da fonte não deveria estar entre aspas simples?
```suggestion
    font-family: 'Poppins', sans-serif;
```